actor/manager/manager.go

```diff
@@ -251,7 +250,7 @@ func suitableMethods(typ reflect.Type) map[string]*MethodType {
 	for m := 0; m < typ.NumMethod(); m++ {
 		method := typ.Method(m)
 		if mt, err := suiteMethod(method); err != nil {
-			log.Printf("method %s is illegal, err = %s, just skip it", method.Name, err)
+			log.Infof("method %s is illegal, err = %s, just skip it", method.Name, err)
```

Comment:
`err` being formatted as a string rather than value (although a nil value should in theory never reach this point).